src/Catel.Analyzers/Extensions/CSharpSyntaxNodeExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Catel.Analyzers
+{
+    using System.Linq;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.CSharp;
+    using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+    public static class CSharpSyntaxNodeExtensions
+    {
+        public static string GetIdentifier(this SyntaxNode syntaxNode)
+        {
+            if (syntaxNode is null)
+            {
+                return null;
+            }
+
+            var identifierSyntax = syntaxNode.ChildNodes().Where(node => node is IdentifierNameSyntax name).FirstOrDefault() as IdentifierNameSyntax;
```

Comment:
As cast without checking for null. But leave as is, I have enabled Nullable on the develop branch so this will fail to compile once we merge anyway.